app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    res_dict = {}
+
+    def inner(*args) -> str:
```

Comment:
```suggestion
    def inner(*args) -> Any:
```
The function returns anything